item.rb

```diff
@@ -0,0 +1,46 @@
+require 'date'
+# Item class
+class Item
+  attr_accessor :publish_date, :archived
+  attr_reader :label, :author, :genre, :source
```

Comment:
- [ optional ] Hi kindly remove the `:source` from attr_reader since its only for teams of 4 and you don't have that table